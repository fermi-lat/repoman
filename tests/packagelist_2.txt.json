[
  ["xmlBase", "xmlBase-05-07-01"],
  ["astro", "astro-04-00-02"],
  ["celestialSources", "celestialSources-01-06-01"],
  ["celestialSources", "genericSources-01-17-01", "genericSources"],
  ["celestialSources", "Pulsar-03-03-00", "Pulsar"],
  ["celestialSources", "eblatten-01-01-01", "eblatten"]
]