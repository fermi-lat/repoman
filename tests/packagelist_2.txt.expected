#  ScienceTools packageList.txt
# $Id: packageList.txt,v 1.211 2017/04/28 19:24:14 jchiang Exp $

# in common with GlastRelease
xmlBase xmlBase-05-07-01
astro astro-04-00-02

# source modeling
celestialSources celestialSources-01-06-01
celestialSources/genericSources genericSources-01-17-01
celestialSources/Pulsar Pulsar-03-03-00
celestialSources/eblatten eblatten-01-01-01
